public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -1433,7 +1433,7 @@ block json-pipe
   ### The safe navigation operator ( ?. ) and null property paths
 
   The Angular **safe navigation operator (`?.`)** is a fluent and convenient way to guard against null and undefined values in property paths.
-  Here it is, protecting against a view render failure if the `currentHero` is null.
+  Here it is used to protect against against a view render failure if the `currentHero` is null:
```

Comment:
I'm a little confused. I believe I just saw this text in a commit that I (belatedly) commented on: https://github.com/angular/angular.io/commit/d0811714d54cda1882f8287e095bf182ce39421d#commitcomment-18338618